source/fundamentals/indexes.txt

```diff
@@ -15,6 +15,8 @@ Indexes
 Overview
 --------
 
+In this guide, you can learn how to use **indexes** in the MongoDB Java driver.
```

Comment:
```suggestion
In this guide, you can learn how to use **indexes** with the MongoDB Java driver.
```

Suggestion: I think this "in" should be "with" as driver does not possess indexes.